pyproject.toml

```
@@ -0,0 +1,9 @@
+[build-system]
+requires = [
+    "setuptools",
```

```suggestion
    "setuptools>=64",
```

as suggested at https://github.com/pypa/pip/issues/11457 - scroll down to What should I do?